### PR TITLE
Add Descriptions to Language Settings

### PR DESCRIPTION
**Pull request by Linfye**
### Contributor checklist

-   [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

I've added a new `description` attribute to `switchItem`, and included descriptions for the current language setting items.
### Related issue

-   #83

### CODE REVIEW

**Comment by andrewtavis** [comment]
Thanks for the PR, @Linfye! Looking forward to checking it out :)

File: app/src/main/java/be/scri/fragments/LanguageSettingsFragment.kt [hunk 1]
```diff
@@ -109,12 +109,14 @@ class LanguageSettingsFragment : Fragment() {
             SwitchItem(
                 isChecked = sharedPref.getBoolean("period_on_double_tap_$language", false),
                 title = "Double space periods",
```
Comment (andrewtavis) [inline app/src/main/java/be/scri/fragments/LanguageSettingsFragment.kt:111]: Checking here in code that wasn't changed, but it also applies to the changed code: Are we using Scribe-i18n for these texts? If not, it'd be great if we start using the localized keys for all app texts :)

**Comment by andrewtavis** [comment]
All's working really well though, @Linfye :) Do you want to look into getting the localized keys in for these two menu options? You could replace their titles and descriptions with the localized Scribe-i18n keys, and then we could make an issue to find and replace the others base strings 🌐

**Comment by Linfye** [comment]
I wasn't aware that there are keys available for these strings. I searched for these words in the current files but couldn't find them. Could you provide more details on how to use these keys? I can replace them accordingly. 😊

**Comment by andrewtavis** [comment]
Ah they might be missing 🤔 I'll add these keys in and then make an issue to convert all the strings! Thanks, @Linfye :)

**Review by angrezichatterbox — APPROVED** [review]
Great PR @Linfye. I am glad to see you continue contributing to Scribe-Android. :)

**Review by andrewtavis — APPROVED** [review]
Approving and will add the keys and strings to Weblate now :) Will make the other issue after 😊

**Comment by andrewtavis** [comment]
All the changes just went through to Scribe-i18n :) @angrezichatterbox, the workflow for Android is failing for some reason. Could you take a look at that?

**Comment by angrezichatterbox** [comment]
> All the changes just went through to [Scribe-i18n](https://github.com/scribe-org/Scribe-i18n) :) @angrezichatterbox, the workflow for Android is failing for some reason. Could you take a look at that?

I'm not able to see the failed workflow. Could I get which workflow it is.

**Comment by andrewtavis** [comment]
It's not in this repo, @angrezichatterbox, but rather in Scribe-i18n: https://github.com/scribe-org/Scribe-i18n/actions/runs/10759643376

**Comment by andrewtavis** [comment]
Let me know if you can't see that!

**Comment by angrezichatterbox** [comment]
> It's not in this repo, @angrezichatterbox, but rather in Scribe-i18n: https://github.com/scribe-org/Scribe-i18n/actions/runs/10759643376

I will look into it.

**Comment by angrezichatterbox** [comment]
I think it is because we need not pass the Scribe-i18n again in the path. 

`python: can't open file '/home/runner/work/Scribe-i18n/Scribe-i18n/Scribe-i18n/scripts/android/convert_jsons_to_strings.py'`

I guess the path for running should be `scripts/android/convert_jsons_to_strings.py`

**Comment by andrewtavis** [comment]
Kind of weird though as they're in `Scribe-i18n/scripts/...`, but let's see if it works :)

**Comment by andrewtavis** [comment]
I think that the big change that needed to happen was to add `./` to the start so it knows the relation to the project root. We'll see in the next commit!

**Comment by angrezichatterbox** [comment]
> I think that the big change that needed to happen was to add `./` to the start so it knows the relation to the project root. We'll see in the next commit!

https://github.com/scribe-org/Scribe-i18n/actions/runs/10400331540/job/28800679978 In this action added it failed right using ./Scripts/....

Run python ./Scripts/Android/convert_jsons_to_strings.py
python: can't open file '/home/runner/work/Scribe-i18n/Scribe-i18n/./Scripts/Android/convert_jsons_to_strings.py': [Errno 2] No such file or directory

**Comment by andrewtavis** [comment]
Ya I saw that and sent a commit along that sets the working directory explicitly :) Hopefully is fine now. Thanks for checking, @angrezichatterbox!

**Pull request merged**